debian/patches/0024-add-sub2video-option-to-subtitles-filter.patch

```diff
@@ -1,8 +1,6 @@
-Index: jellyfin-ffmpeg/libavfilter/vf_subtitles.c
-===================================================================
```

Comment:
LTGM. But any reason why indexes are stripped?

Reply:
I've finally started using quilt, and just pasted the .quiltrc from here: https://wiki.debian.org/UsingQuilt
I can redo it without QUILT_NO_DIFF_INDEX=1

Reply:
Thanks. I'll merge this first so as you can add `fdc_aac` to the debian changelog in another PR.